GitUI/Editor/FileViewer.cs

```diff
@@ -579,17 +579,7 @@ public Task ViewGrepAsync(FileStatusItem item, string text)
                     {
                         try
                         {
-                            StringBuilder summary = new StringBuilder()
-                                .AppendLine("Binary file:")
-                                .AppendLine()
-                                .AppendLine(fileName)
-                                .AppendLine()
-                                .AppendLine($"{text.Length:N0} bytes:")
-                                .AppendLine();
-                            internalFileViewer.SetText(summary.ToString(), openWithDifftool);
-
-                            ToHexDump(Encoding.ASCII.GetBytes(text), summary);
-                            internalFileViewer.SetText(summary.ToString(), openWithDifftool);
+                            DisplayAsHexDump("Binary file: " + fileName, text, openWithDifftool);
```

Comment:
```suggestion
                            DisplayAsHexDump("Binary file: {0}", fileName, text, openWithDifftool);
```
as translated string?